### ISSUE TITLE
L1B parameters missing in measurement granularity

### ISSUE THREAD

**Issue by paoladaulerio** [ESA-VirES/Aeolus-Client#52]

**Comment by paoladaulerio** [comment]
When some parameters in L1b (e.g. Mie Signal intensity, Rayleigh Channel A/B Signal Intensity) are selected in "measurements" granularity  the curtain is not visualized on the globe
Same parameters are selectable in the analytic view

**Comment by santilland** [comment]
Thank you for the info. It seems the intensity parameter is not fetched correctly for measurements. 
If i am not missing something the parameter should not be selectable in the analytics viewer either.
Will further investigate.

**Comment by paoladaulerio** [comment]
Correct, the right sentence in my comment above is "Same parameters are not selectable in the analytic view"

**Comment by santilland** [comment]
Parameters are now visible and can be selected also for measurements, will close this ticket if there are no objections.
